index.html

```diff
@@ -8,6 +8,69 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+    <div id="blue-box">
+        <div id="join-us">
+            <div id="subscribe">
+                    <div class="msg">Subscribe to get information, latest news and other</div>
+                    <div class="msg">interesting offers about Cobham</div>
+            </div>
+            <div id="mail">
+                <div id="mail-bar">
+                        <img id="mail-image" src="./images/email.svg">
+                        <div id="sub-gap">Your email</div>
+                </div>
+                <div id="button">
+                    <div id="green">Subscribe</div>
+                </div>
+            </div>
+        </div>
+        <img id="telegram" src="./images/tele.svg" />
+    <!-- <img id="stars" src="./images/stars.svg" /> -->
+    </div>
+    <div id="second-half">
+        <div id="social-media">
```

Comment:
social-media is not a proper id. Since, if you look at the content of this division it has lot of content that is not related to social media.